src/modules/logger/logger.cpp

```diff
@@ -2040,6 +2042,10 @@ void Logger::write_version(LogType type)
 		write_info(type, "sys_os_ver", os_version);
 	}
 
+	if (git_tag && git_tag[0]) {
+	    write_info(type, "ver_git_tag", git_tag);
```

Comment:
I suppose to keep the naming consistent this should probably be `ver_sw_tag`. The branch (git_branch) is `ver_sw_branch`.

```suggestion
	    write_info(type, "ver_sw_tag", git_tag);
```